ScarletEngine/src/AssetManager/AssetHandle.cpp

```diff
@@ -50,4 +50,95 @@ namespace ScarletEngine
 		stbi_image_free(PixelDataBuffer);
 	}
 
+	// #todo: move this somewhere else
+	static void LoadObj(const String& FilePath, Array<Vertex>& OutVertices, Array<uint32_t>& OutIndices)
```

Comment:
future task could be to support Assimp so we can load more file types. Also doesn't seem like materials are loaded? another future tasks we can add